scripts/gc.sh

```diff
@@ -30,50 +30,58 @@ mkdir -p "$DIRNAME"
 mkdir -p "$DIRNAME/src"
 mkdir -p "$DIRNAME/__tests__"
 
-cat > $DIRNAME/src/index.vue <<EOF
+cat > $DIRNAME/src/$INPUT_NAME.vue <<EOF
 <template>
   <div>
-    <slot></slot>
+    <slot />
   </div>
 </template>
-<script lang='ts'>
-import { defineComponent } from 'vue'
-export default defineComponent({
-  name: 'El${NAME}',
-  props: { },
-  setup(props) {
-    // init here
-  },
+<script lang="ts" setup>
```

Comment:
I think it would be better to insert a blank line between `template` and `script` too.